robotiq_hande_gripper_description/urdf/robotiq_hande.urdf

```diff
@@ -83,7 +83,10 @@
     <selfCollide>True</selfCollide>
   </gazebo>
   <gazebo>
-    <plugin filename="librobotiq_linear_gripper_gazebo_plugin.so" name="robotiq_hande_gripper">
+    <plugin filename="librobotiq_gripper_gazebo_plugin.so" name="robotiq_hande_gripper">
+      <p>10</p>
```

Comment:
I would call this `ki`, `kp` and `kd`

Reply:
https://github.com/AcutronicRobotics/robotiq_modular_gripper/pull/2/commits/1e1314ecce4bf52483e366a87910af57a9017bd6